services/movie_session.py

```diff
@@ -41,4 +42,15 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    try:
+        MovieSession.objects.get(id=session_id).delete()
+    except ObjectDoesNotExist as e:
+        raise ValidationError(e)
```

Comment:
Raising a `ValidationError` with the caught exception message might not be the best approach. Consider providing a more user-friendly error message to indicate that the movie session could not be found for deletion.